inlong-sort/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/multiple/RecordWithSchema.java

```diff
@@ -56,6 +58,16 @@ public RecordWithSchema(
         this.primaryKeys = primaryKeys;
     }
 
+    @Setter
```

Comment:
can be replaced by @Data annotation on RecordWithSchema?

Reply:
Fixed